src/pipemode_op/RecordReader/RecordIOReader.cpp

```diff
@@ -64,8 +66,8 @@ bool RecordIOReader::ReadRecord(std::string* storage) {
         std::size_t expected_size = GetRecordSize(header);
         std::size_t padded_expected_size = GetPaddedSize(expected_size);
         total_record_size += expected_size;
-        storage->resize(total_record_size);
-        Read(&(storage->at(total_record_size - expected_size)), expected_size);
+        storage->resize_uninitialized(total_record_size);
+        Read(&((*storage)[total_record_size - expected_size]), expected_size);
```

Comment:
Is at() function deprecated?

Reply:
tensorflow::tstring does not implement at()